app/uk/gov/hmrc/helptosave/util/HttpResponseOps.scala

```diff
@@ -45,7 +45,9 @@ class HttpResponseOps(val response: HttpResponse) extends AnyVal {
       couldntParseJson = (_, e: JsError) => s"Could not parse http response JSON: ${e.prettyPrint()}"
     )
 
-  def desCorrelationId: String = response.header("CorrelationId").getOrElse("-")
+  def desCorrelationId: String = response.header("DesCorrelationId").getOrElse("-")
```

Comment:
I believe the correlation ids should both be "CorrelationId"

Reply:
Yes done